massa-execution/src/vm.rs

```diff
@@ -151,8 +148,12 @@ impl VM {
                 .unwrap(),
             true,
         );
-        // Save the Initial ledger changes before execution
-        // It contains a copy of the initial coin credits that will be popped back if bytecode execution fails in order to cancel its effects
+
+        // credit the sender with "coins"
```

Comment:
nit: I'd rather have upper case at the start of each sentence.

Reply:
done